src/main/java/cn/nukkit/block/BlockIcePacked.java

```diff
@@ -33,8 +34,12 @@ public boolean canHarvestWithHand() {
     
     @Override
     public boolean onBreak(Item item) {
-        this.getLevel().setBlock(this, Block.get(AIR), true); //no water
-        return true;
+        return this.getLevel().setBlock(this, Block.get(AIR), true); //no water
+    }
+
+    @Override
+    public boolean onBreak(Item item, Player player) {
+        return this.onBreak(item);
```

Comment:
This is to override the default Ice behaviour, because packed ice always becomes air when broken